doc/sphinx/source/tutorials/apfelcomb.rst

```diff
@@ -0,0 +1,362 @@
+scattering Reduced Cross-Section, light (electron-proton) - DIS_CCP_L:
+[CC] Positron scattering Reduced Cross-Section, light (positron-proton)
+- DIS_CCE_C: [CC] Electron scattering Reduced Cross-Section, charm
+(electron-proton) - DIS_CCP_C: [CC] Positron scattering Reduced
+Cross-Section, charm (positron-proton) - DIS_CCE: [CC] Electron
+scattering Reduced Cross-Section, inclusive (electron-proton) - DIS_CCP:
+[CC] Positron scattering Reduced Cross-Section, inclusive
+(positron-proton)
+
+**Deep Inelastic Scattering Reduced Cross-Sections (heavy-ion)**: -
+DIS_SNU_L_Pb: [CC] Neutrino scattering Reduced Cross-Section, light
+(neutrino-lead) - DIS_SNB_L_Pb: [CC] Antineutrino scattering Reduced
+Cross-Section, light (antineutrino-lead) - DIS_SNU_C_Pb: [CC] Neutrino
+scattering Reduced Cross-Section, charm (neutrino-lead) - DIS_SNB_C_Pb:
+[CC] Antineutrino scattering Reduced Cross-Section, charm
+(antineutrino-lead) - DIS_SNU_Pb: [CC] Neutrino scattering Reduced
+Cross-Section, inclusive (neutrino-lead) - DIS_SNB_Pb: [CC] Antineutrino
+scattering Reduced Cross-Section, inclusive (antineutrino-lead) -
+DIS_SNU_L: [CC] Neutrino scattering Reduced Cross-Section, light
+(neutrino-isoscalar) - DIS_SNB_L: [CC] Antineutrino scattering Reduced
+Cross-Section, light (antineutrino-isoscalar) - DIS_SNU_C: [CC] Neutrino
+scattering Reduced Cross-Section, charm (neutrino-isoscalar) -
+DIS_SNB_C: [CC] Antineutrino scattering Reduced Cross-Section, charm
+(antineutrino-isoscalar) - DIS_SNU: [CC] Neutrino scattering Reduced
+Cross-Section, inclusive (neutrino-isoscalar) - DIS_SNB: [CC]
+Antineutrino scattering Reduced Cross-Section, inclusive
+(antineutrino-isoscalar) - DIS_DM_NU: [CC] Dimuon neutrino cross section
+(neutrino-iron) - DIS_DM_NB: [CC] Dimuon anti-neutrino cross section
+(antineutrino-iron)
+
+**Single-Inclusive electron-positron annihilation, Time-Like Evolution
+(SIA)**: - SIA_F2: [NC] SIA structure function F2 = FT + FL
+(electron-proton) - SIA_FL: [NC] SIA structure function FL
+(electron-proton) - SIA_FA: [NC] SIA structure function FA
+(electron-proton) - SIA_XSEC_NF4: [NC] SIA absolute cross section (nf=4)
+(electron-proton) - SIA_XSEC: [NC] SIA absolute cross section
+(electron-proton) - SIA_NORM_XSEC_LONG_L: [NC] SIA normalized light
+longitudinal cross section (electron-proton) - SIA_NORM_XSEC_LONG_BT:
+[NC] SIA normalized bottom longitudinal cross section (electron-proton)
+- SIA_NORM_XSEC_LONG: [NC] SIA normalized total longitudinal cross
+section (electron-proton) - SIA_NORM_XSEC_L: [NC] SIA normalized light
+cross section (electron-proton) - SIA_NORM_XSEC_CH: [NC] SIA normalized
+charm cross section (electron-proton) - SIA_NORM_XSEC_BT: [NC] SIA
+normalized bottom cross section (electron-proton) - SIA_NORM_XSEC_TP:
+[NC] SIA normalized top cross section (electron-proton) -
+SIA_NORM_XSEC_NF4: [NC] SIA normalized total cross section (nf=4)
+(electron-proton) - SIA_NORM_XSEC: [NC] SIA normalized total cross
+section (electron-proton)
```

Comment:
The conversion has not quite worked for this list (adding `\n` before every `-` should suffice I guess?)